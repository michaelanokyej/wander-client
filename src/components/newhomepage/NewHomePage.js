import React from "react";
import TourSearch from '../landingpage/tour-search/TourSearch'
import HomeTourList from '../landingpage/home-tour-list/HomeTourList'
import Footer from '../landingpage/footer/Footer'

class NewHomePage extends React.Component {
  render() {
    return (
      <main>
        <header role="banner">
			  <h1>Wander</h1>
        <h4>Tour like a local</h4>
		    </header>
        <TourSearch />
        <HomeTourList />
        <Footer />
      </main>
    );
  }
}

export default NewHomePage;
