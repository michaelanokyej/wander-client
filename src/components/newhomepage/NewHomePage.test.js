import React from "react";
import NewHomePage from "./NewHomePage"
import { shallow } from "enzyme"
import toJson from "enzyme-to-json"

describe(`NewHomePage component`, () => {
    it(`renders the NewHomePage component`, () => {
        const wrapper = shallow(<NewHomePage />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});