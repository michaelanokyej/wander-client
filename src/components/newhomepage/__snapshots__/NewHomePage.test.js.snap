// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewHomePage component renders the NewHomePage component 1`] = `
<main>
  <TourSearch />
  <HomeTourList />
</main>
`;
